app/controllers/questions_controller.rb

```diff
@@ -6,7 +6,10 @@ def index
   end
 
   def show
-    @question = get_question(quesiton_id)
+    @question = get_question(show_params[:id])
+
+    # アクセス権限がない場合は、404を返す
+    render json: { errors: { message: 'ページが見つかりません(アクセス権限なし)' } }, status: :not_found if @question.nil?
```

Comment:
- アクセス権限がない場合は、404を返す ことは見ればわかる
- アクセス権限がない場合だけではなさそう

```suggestion
     render_not_found if @question.nil?
```

```
def render_not_found
  render json: { errors: { message: 'ページが見つかりません(アクセス権限なし)' } }, status: :not_found
```